src/controller/python/templates/python-cluster-Objects-py.zapt

```diff
@@ -105,6 +106,12 @@ class {{asUpperCamelCase name}}(Cluster):
                 return ClusterObjectFieldDescriptor(Type={{zapTypeToPythonClusterObjectType type ns=(asUpperCamelCase parent.name)}})
             {{/if}}
 
+            {{#if entryType}}
+            value: '{{zapTypeToPythonClusterObjectType entryType ns=(asUpperCamelCase parent.name)}}' = None
```

Comment:
please use real type or put the type as a comment - I don't think a  string is a valid type hint

Reply:
Had side conversation  with @mrjerryjohns - these types are shown by 'help' so even if the string is wrong, it is usefull.

Please create an GH issue for it and we can merge as is until we figure out correct typing. Long term however, I belive we should use comments (or doc comments if needed and make help use that) or real types rather than strings.

Reply:
#12062